.gitignore

```diff
@@ -3,3 +3,4 @@
 .vscode/
 gokoala
 assets/vectortile-view-component/
+webcomponents/vectortile-view-component/src/app/openapi
```

Comment:
De naam "vectortile-view-component" dekt de lading niet helemaal meer. Het is 1 angular codebase met daarin meerdere webcomponents. Ik stel voor dat we het hernoemen:

- vectortile-view-component => viewer (vrij algemene naam maar het is de viewer binnen de context van gokoala, geoviewer of webviewer zouden alternatieven kunnen zijn)
- webcomponents directory verwijderen en viewer als directory direct in de root plaatsen. Dus `<gokoala root>webcomponents/vectortile-view-component/` wordt dan `<gokoala root>/viewer`.

Evt kan ik dit ook oppakken als je dat liever wilt.

Reply:
Ik heb het een en ander aangepast. Bij upgraden moeten we wel realiseren dat "afwijkende" go HTML templates b.v. de BGT misschien aangepast moeten worden. Waarschijnlijk staat hierin de harde link: 

<script type="text/javascript" src="vectortile-view-component/main.js"></script>
i.p.v. 
<script type="text/javascript" src="view-component/main.js"></script>